types/node/tls.d.ts

```diff
@@ -64,70 +64,72 @@ declare module "tls" {
         version: string;
     }
 
+    export interface TLSSocketOptions {
```

Comment:
I think `enableTrace?: boolean` should be added here also.

Reply:
Not according to the changelog, this is just cleanup.

Reply:
but according to the docs: https://nodejs.org/dist/latest-v12.x/docs/api/tls.html#tls_new_tls_tlssocket_socket_options and https://nodejs.org/dist/latest-v12.x/docs/api/tls.html#tls_tls_connect_options_callback

Reply:
*sigh* I wish the changelog was more detailed.